src/nrnpython/CMakeLists.txt

```diff
@@ -225,10 +226,6 @@ if(NRN_ENABLE_MODULE_INSTALL)
       POST_BUILD
       COMMAND ${CMAKE_COMMAND} -E copy_if_different ${CMAKE_CURRENT_SOURCE_DIR}/inithoc.cpp
               ${CMAKE_CURRENT_BINARY_DIR}/inithoc.cpp
-      COMMAND
-        ${CMAKE_COMMAND} -E copy_if_different
-        ${PROJECT_SOURCE_DIR}/share/lib/python/neuron/help_data.dat
-        ${CMAKE_CURRENT_BINARY_DIR}/lib/python/neuron/help_data.dat
```

Comment:
Originally I guess there was no reason to have this over all python executables that we build with (i.e. help_data.dat doesn't get augmented).

Reply:
Yes, I vaguely remember having issues with packaging the `dat` file, maybe older distutils/setuptools had some quirks.